requirements.txt

```diff
@@ -1,6 +1,5 @@
-flask
-gym
```

Comment:
I'm unsure of all the places we use this rather than the list in `setup.py` but did you verify this doesn't break the docker build and tests?